status/k8s.go

```diff
@@ -357,13 +358,28 @@ func (k *K8sStatusCollector) Status(ctx context.Context) (*Status, error) {
 		}
 		if !k.statusIsReady(s) && k.params.Wait {
 			time.Sleep(defaults.WaitRetryInterval)
+			if k.params.Output != "json" {
+				statusFmt := s.Format()
+				cursorUp(lines)
+				lines = len(strings.Split(statusFmt, "\n"))
+				fmt.Print(statusFmt)
+			}
 			continue
 		}
 
+		for i := 1; i < lines; i++ {
+			fmt.Print("\033[A\033[2K")
```

Comment:
How does this sequence fare on different platforms?

Reply:
It should be pretty safe on Linux and MacOS. On Windows, it ANSI codes seem to be well supported for Windows 10+.